src/test/java/com/gooddata/auditevent/AuditEventServiceTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ */
+package com.gooddata.auditevent;
+
+import com.gooddata.account.Account;
+import com.gooddata.account.AccountService;
+import com.gooddata.collections.PageRequest;
+import org.springframework.web.client.RestTemplate;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class AuditEventServiceTest {
+
+    private AuditEventService service;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        service = new AuditEventService(new RestTemplate(), mock(AccountService.class));
```

Comment:
To be honest, I expect this test to be more thorough... Mock the REST template and verify the executed URIs to begin with.

Reply:
already covered by IT

Reply:
Well at least the paging part is not tested at all. I leave it up to your self-confidence :)

Reply:
added test for paging to IT